### ISSUE TITLE
View [layouts.app] not found in laravel

### ISSUE THREAD

**Issue by abdullahicyc** [bradtraversy/lsapp#2]
These are my views
![image](https://user-images.githubusercontent.com/15052571/34568260-4ac6ecec-f176-11e7-816f-978bc767c6db.png)

and this is my app.blade.php file
![image](https://user-images.githubusercontent.com/15052571/34568341-87dff0d8-f176-11e7-94ec-b8810e53eec7.png)

so when i visit the index page, i get this error 
![image](https://user-images.githubusercontent.com/15052571/34568432-d4f79cb8-f176-11e7-90bb-1c30ed6dea0a.png)

TL;DR -I'm following this serious.

**Comment by QuentinBultinck** [comment]
# Solution 1
In your index.blade.php replace `@extends('layouts.app')` 
with `@extends('pages.layouts.app')`

# Solution 2
Place your layouts folder in the root of your view folder.
So it should look something like this:

![afbeelding](https://user-images.githubusercontent.com/17977577/34602557-82e86210-f200-11e7-9ee9-2259e86a63be.png)

# Info
`@extends('layouts.app')`
Always starts searching at the root of the views folder.

**Comment by abdullahicyc** [comment]
thanks @bultinckquentin 
its working now.

**Comment by ernomanesh** [comment]
Hello,I have tried to attach mine to the root folder but still is giving a 404 error,how can you help me on that.

**Comment by Tazuba** [comment]
solution 1 worked fine for mi.
![blade](https://user-images.githubusercontent.com/42208290/82741687-8cebbb80-9d5d-11ea-9ec5-f5590c357ada.JPG)

**Comment by Justin-ic** [comment]
Good evening everyone!!

If you write App.blade.php and you use @extends('layouts.app'), it can work in local with Laravel 8 but 
in production, such as Heroku, you must write app.blade.php and use @extends('layouts.app')
